articles/openshift/oshift-trials-gs.md

```diff
@@ -0,0 +1,67 @@
+
+With UKCloud OpenShift you have admin access to OpenShift giving you the control you need to create, deploy and administar your cluster as you see fit.
+
+All OpenShift deployments get a starter pack which includes the management and orchestration planes of OpenShift. These are essentially 3 Master, 2 Infra, 2 data and 2 control plane nodes.
+
+During a trial each cluster is also provisioned with 2 small run time packs. These are essentially 2 OpenStack VM's running RedHat atomic OS with 16GB RAM, 2vCPU's and 100GB disks each. This is also the minimum number of Runtime Packs UKCloud recommends in each OpenShift deployment. After the trial phase it is possible to deploy larger and more nodes for your cluster.
+
+**OpenShift documentation:** [*Architecture*](https://docs.openshift.com/container-platform/3.11/architecture/index.html)
+
+## GUI and CLI Access
+
+In order to access the OpenShift cluster GUI customers require a internet-enabled device running a HTML5 capable browser and need to be set up for access in UKClouds Portal with SSO access. Customers will also be set up with the relavent access in their OpenShift deployment.
+
+**OpenShift documentation:** [*Web Console Walkthrough*](https://docs.openshift.com/container-platform/3.11/getting_started/developers_console.html)
+
+In addition to GUI access customers can also access OpenShift through the CLI, in order to do this customers need to have installed the OC client on their machines. 
+
+**OpenShift documentation:** [*Installing the OC CLI*](https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html#installing-the-cli)
+
+## Storage
+
+Each trial deployment has a block storage limit allocated to the OpenStack project of 2TB. The storage is available to the OpenShift projects customers deploy within OpenShift, customers can consume that storage by creating Persistant Storage Claims for there applications. It is good practice to use PVC's for any persistant storage requirements as any data stored within pods will be lost when clusters are patched. After trials if customers require more than 2TB of storage to be available to there clusters than customers can request this. 
+
+OpenShift is deployed with an internal image registry with a 60GB capacity. Customers can make use of this registry which is backed by UKClouds object storage. This does not come out of customers 2TB storage capacity, rather this is in addition to customers 2TB data limit. However UKCloud do provision ElasticSearch, FluentD and Kibana stack which uses 20GB and Hawkular Metrics backed by a Cassandra database which uses 10GB and the Prometheus Operator which uses 106GB of tier 2 storage which is taken out of the 2TB storage limit.  
```

Comment:
```suggestion
OpenShift is deployed with an internal image registry making use of UKClouds object storage service for persistence.  This includes 60GB of free storage capacity and does not come out of customers 2TB block storage capacity, rather this is in addition to customers 2TB data limit. However UKCloud do provision ElasticSearch, FluentD and Kibana stack which uses 20GB and Hawkular Metrics backed by a Cassandra database which uses 10GB and the Prometheus Operator which uses 106GB of tier 2 storage which is taken out of the 2TB storage limit.  
```

Reply:
It'd be worth breaking out the individual persistent volumes and their sizes in to a list here as well, so that its clear how much each is using.